src/index.html

```diff
@@ -11,6 +11,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="container">
+      <form
+        class="form__search"
+        action="#"
+        method="post"
+        data-qa="big"
+      >
+        <input
+            class="first__inp input__bar"
+            type="search"
+            name="big-search"
+            placeholder="Try &quot;Los Angeles&quot;"
+            data-qa="keypress"
+        >
```

Comment:
```suggestion
        <input
          class="first__inp input__bar"
          type="search"
          name="big-search"
          placeholder="Try &quot;Los Angeles&quot;"
          data-qa="keypress"
        >
```
![image](https://user-images.githubusercontent.com/96409563/222386122-eb3acbe0-f817-46e7-a5c9-f28afad62441.png)

There should be only 2 spaces of indentation. Please fix all cases